tutorials/introductory/getting_started.py

```diff
@@ -0,0 +1,767 @@
+
+plt.show()
+
+##############################################################################
+#
+#
+
+plt.pie(budget, colors=colors, labels=categories)
+plt.legend()
+plt.title('Average Monthly Income Expenses')
+plt.axis('equal')
+# The pyplot module contains an identical method for aspect ratio setting.
+
+plt.show()
+
+##############################################################################
+#
+# .. note::
+#
+#   There are minor differences in the method names. Overall, each method
+#   performs the same action through the different approaches.
+#
+# These pie charts are simple in that there is not much distinguishing
+# information. Keyword arguments and Artists add the ability to implement
+# more ways of displaying content.
+#
+# Additional Customization
+# ^^^^^^^^^^^^^^^^^^^^^^^^
+#
+# Many methods contain optional keyword arguments for further configuration.
+# In the examples for explicit programming below, there are values and
+# functions in keyword arguments that format the Artists. These changes also
+# apply to implicit programming, though with varying method names.
+#
+# The pie chart below adds configurations with keyword arguments for
+# ``explode``, ``autopct``, ``startangle``, and ``shadow``. These keyword
+# arguments help to manipulate the Artists.
+
+# Explicit
+
+fig, ax = plt.subplots()
+
+ax.pie(budget,
+       colors=colors,
+       labels=categories,
+       explode=explode,
+       # The explode keyword argument uses the explode variable from data to
+       # separate respective wedges from the center.
+       autopct='%1.1f%%',
+       # The autopct keyword argument takes formatting strings and functions
+       # to generate text within the wedge. '%1.1f%%' uses string formatters.
+       startangle=-80,
+       # The startangle keyword argument changes where the first wedge spans.
+       # Angles start at 0 degrees on the X-axis and move counterclockwise.
+       shadow=True
+       # The shadow keyword argument toggles a shadow on the visual.
+       )
+
+ax.legend()
+ax.set_title('Average Monthly Income Expenses')
+ax.axis('equal')
+
+plt.show()
+
+##############################################################################
+#
+# In the pie chart below, there are additional keyword arguments to further
+# customize the visual. Also, the ``legend`` as an Artist has parameters that
+# enable more specification for the information displayed. For more, see the
+# :ref:`legend-guide`
+
+# Explicit
+
+fig, ax = plt.subplots()
+
+budget_pie = ax.pie(budget,
+                    colors=colors,
+                    labels=descriptions,
+                    # Instead of using the categories data as a label, the
+                    # descriptions act as text to label the wedges. This aids
+                    # in removing redundant information from the previous
+                    # pie chart.
+                    explode=explode,
+                    autopct=lambda pct: autopct_format(pct, budget),
```

Comment:
either write a short comment explaining that this binds the budget variable so inside the pie function autopct_format is used w/ code inside pie providing the pct values.
autopct=functools.partial(auto_pct_format, budget=budget)
https://docs.python.org/3/library/functools.html